sim/shaman/enhancement/enhancement.go

```diff
@@ -109,10 +109,36 @@ func (enh *EnhancementShaman) Initialize() {
 		return spell.MeleeAttackPower() * 0.55
 	}
 
+	// Mastery: Enhanced Elements
+	masteryMod := enh.AddDynamicMod(core.SpellModConfig{
+		Kind:       core.SpellMod_DamageDone_Pct,
+		FloatValue: enh.getMasteryBonus(),
```

Comment:
You dont have all mastery stats calculated at this point (Missing consumes/buffs). Instead just dont assign a float value here and assign it in the OnGain of the mastery aura